src/router/PrefixMatching/PrefixMap.php

```diff
@@ -73,23 +77,28 @@ private static function fromFlatMapImpl(
       if ($node is ParameterNode && $node->getRegexp() === null) {
         $next = C\first($nodes);
         if (
-          $next is LiteralNode && Str\starts_with($next->getText(), '/')
+          $next is LiteralNode &&
+          $next->getText() &&
```

Comment:
Going to merge but will be changing this part: this is a generic falsiness test which:
- we are aiming to remove for strings and consider an antipattern
- is unneccessary here